control-plane-operator/controllers/hostedcontrolplane/testdata/cluster-version-operator/zz_fixture_TestControlPlaneComponents_TechPreviewNoUpgrade.yaml

```diff
@@ -129,56 +129,56 @@ spec:
+          rm -f /var/payload/release-manifests/0000_90_olm_00-service-monitor.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_04_service_account.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_05_role.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_06_role_binding.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_07_configmap.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_08_service.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_09_operator-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_09_operator.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_10_clusteroperator.yaml
+          rm -f /var/payload/release-manifests/0000_50_operator-marketplace_11_service_monitor.yaml
+          rm -f /var/payload/release-manifests/0000_70_dns-operator_02-deployment-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-ingress-operator_02-deployment-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_80_machine-config_01_containerruntimeconfigs.crd.yaml
+          rm -f /var/payload/release-manifests/0000_80_machine-config_01_kubeletconfigs.crd.yaml
+          rm -f /var/payload/release-manifests/0000_80_machine-config_01_machineconfigs.crd.yaml
+          rm -f /var/payload/release-manifests/0000_80_machine-config_01_machineconfigpools-Default.crd.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-node-tuning-operator_20-performance-profile.crd.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-node-tuning-operator_50-operator-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-image-registry-operator_07-operator-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-image-registry-operator_07-operator-service.yaml
+          rm -f /var/payload/release-manifests/0000_90_cluster-image-registry-operator_02_operator-servicemonitor.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-storage-operator_10_deployment-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_50_cloud-credential-operator_01-operator-config.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-authentication-operator_02_config.cr.yaml
+          rm -f /var/payload/release-manifests/0000_90_etcd-operator_03_prometheusrule.yaml
+          rm -f /var/payload/release-manifests/0000_50_cluster-csi-snapshot-controller-operator_07_deployment-ibm-cloud-managed.yaml
+          rm -f /var/payload/release-manifests/0000_03_marketplace-operator_02_operatorhub.cr.yaml
```

Comment:
thanks for fixing this so quickly @sjenning - maybe it's worth adding a test over this file to check if there are any "rm" without "-f"?